.github/config/android-x86.json

```diff
@@ -19,6 +19,9 @@
         "system_webview_shell_apk",
         "cobalt:gn_all"
     ],
+    "test_on_device": {
```

Comment:
Could this be a boolean value instead? I don't see us adding anything else in this object.

Reply:
Wouldn't we be adding additional parameter like we do in https://github.com/youtube/cobalt/blob/25.lts.1%2B/.github/config/android-arm64.json#L3 ?

Reply:
Test suites can be in a separate field. I don't think it should be nested under here. Same with sharding, which will have to be keyed on test suite.

Reply:
done